empress/recon_vis/tanglegram.py

```diff
@@ -138,15 +137,15 @@ def _render_helper_parasite(fig, node, show_internal_labels):
         # plot node using node_layout
         current_loc = (node.layout.col, node.layout.row)
         if show_internal_labels:
-            fig.text(current_loc, node.name, col=plot_tools.BLUE, h_a='right')
+            fig.text(current_loc, node.name, col=render_settings.BLUE, h_a='right')
 
         # draw line from current node to left node
         left_loc = (left_layout.col, left_layout.row)
-        fig.line(current_loc, (node.layout.col, left_layout.row), col=plot_tools.BLACK)
-        fig.line((node.layout.col, left_layout.row), left_loc, col=plot_tools.BLACK)
+        fig.line(current_loc, (node.layout.col, left_layout.row), col=render_settings.BLACK)
+        fig.line((node.layout.col, left_layout.row), left_loc, col=render_settings.BLACK)
 
         # draw line from current node to right node
         right_loc = (right_layout.col, right_layout.row)
-        fig.line(current_loc, (node.layout.col, right_layout.row), col=plot_tools.BLACK)
-        fig.line((node.layout.col, right_layout.row), right_loc, col=plot_tools.BLACK)
+        fig.line(current_loc, (node.layout.col, right_layout.row), col=render_settings.BLACK)
+        fig.line((node.layout.col, right_layout.row), right_loc, col=render_settings.BLACK)
 
```

Comment:
This file as one *extra* newline at the end of file. Please remove this newline.

Reply:
fixed

Reply:
fixed